explicitly provide memory format when calling to clone() at batchnorm.py

ifedan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #28688 explicitly provide memory format when calling to clone() at ProcessGroupGloo.cpp
* #28702 explicitly provide memory format when calling to clone() at SparseCUDATensorMath.cu
* #28701 explicitly provide memory format when calling to clone() at SparseCUDATensor.cpp
* #28700 explicitly provide memory format when calling to clone() at SparseTensorMath.cpp
* #28699 explicitly provide memory format when calling to clone() at SparseTensor.cpp
* #28698 explicitly provide memory format when calling to clone() at TensorShape.cpp
* #28697 explicitly provide memory format when calling to clone() at SparseTensorUtils.h
* #28694 explicitly provide memory format when calling to clone() at observer.py
* #28693 explicitly provide memory format when calling to clone() at rprop.py
* #28692 explicitly provide memory format when calling to clone() at lbfgs.py
* #28691 explicitly provide memory format when calling to clone() at spectral_norm.py
* #28690 explicitly provide memory format when calling to clone() at parameter.py
* **#28689 explicitly provide memory format when calling to clone() at batchnorm.py**
* #28660 explicitly provide memory format when calling to clone() at Indexing.cpp

Differential Revision: [D18333368](https://our.internmc.facebook.com/intern/diff/D18333368)

ifedan committed:
explicitly provide memory format when calling to clone() at batchnorm.py

[ghstack-poisoned]

ifedan committed:
Update on "explicitly provide memory format when calling to clone() at batchnorm.py"




[ghstack-poisoned]

VitalyFedyunin reviewed (commented):
We need preserve here

ifedan committed:
Update on "explicitly provide memory format when calling to clone() at batchnorm.py"


Differential Revision: [D18333368](https://our.internmc.facebook.com/intern/diff/D18333368)

[ghstack-poisoned]

ifedan commented:
> We need preserve here

Changed to Preserve

ifedan committed:
Update on "explicitly provide memory format when calling to clone() at batchnorm.py"


Differential Revision: [D18333368](https://our.internmc.facebook.com/intern/diff/D18333368)

[ghstack-poisoned]

ifedan committed:
Update on "explicitly provide memory format when calling to clone() at batchnorm.py"


Differential Revision: [D18333368](https://our.internmc.facebook.com/intern/diff/D18333368)

[ghstack-poisoned]

ifedan committed:
Update on "explicitly provide memory format when calling to clone() at batchnorm.py"


Differential Revision: [D18333368](https://our.internmc.facebook.com/intern/diff/D18333368)

[ghstack-poisoned]
